.github/actions/cargo-sweep/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cargo-sweep-action",
```

Comment:
Why do we need this to be a JS action?

Reply:
Because the only way to run a `post` step is with a js action afaik